src/DotNet.Consolidate/Models/Options.cs

```diff
@@ -6,14 +6,15 @@ namespace DotNet.Consolidate.Models
 {
     public class Options
     {
-        public Options(ICollection<string>? solutions, ICollection<string>? packageIds, ICollection<string>? excludedPackageIds, string excludedPackageVersionsRegex, bool readDirectoryBuildProps, bool reportOverridenDirectoryBuildProps)
+        public Options(ICollection<string>? solutions, ICollection<string>? packageIds, ICollection<string>? excludedPackageIds, string excludedPackageVersionsRegex, bool readDirectoryBuildProps, bool reportOverridenDirectoryBuildProps, bool useJson)
```

Comment:
Let's use `enum` for the output format option.
Like `text`, `json`, and other formats.

Reply:
Sure.